cartography/data/jobs/cleanup/aws_import_identity_center_cleanup.json

```diff
@@ -0,0 +1,36 @@
+{
+  "statements": [
+    {
+      "query": "MATCH (n:AWSIdentityCenterInstance) WHERE n.lastupdated <> $UPDATE_TAG WITH n LIMIT $LIMIT_SIZE DETACH DELETE (n) RETURN COUNT(*) as TotalDeleted",
```

Comment:
Since the sync job runs one account at a time, this will currently delete all other AWSIdentityCenterInstance nodes in other accounts when we actually want to only delete the stale ones in the current account.

So since you've already built a data mode for this, you don't need to have a manually written cleanup job anymore and you can just do something like this:

https://github.com/cartography-cncf/cartography/blob/bf8d6670a9687bfb5dc40776af37b8a30cf2c910/cartography/intel/aws/ec2/instances.py#L307-L310

It will automatically generate and run the cleanup queries for you so you don't have to worry about any mistakes.

Reply:
Smart move. I will update